.changes/next-release/Bug Fix-07e700f5-e536-43f7-8eee-c95d61158bf6.json

```diff
@@ -0,0 +1,4 @@
+{
+    "type": "Bug Fix",
+    "description": "Added docker network option support for deploying sam applications"
```

Comment:
Please change `deploying` to `invoking`, to better articulate that it is used when you are locally running/debugging your SAM Application instead of pushing it to your AWS account.

Reply:
Will do.